visualizer/actionButton.js

```diff
@@ -279,4 +366,17 @@
     document.querySelector('#operations').innerHTML = '';
     document.querySelector('#attributes').innerHTML = '';
   });
+
+  // Hide the action context menu when Esc or Enter is pressed, any click happens, or another
+  // context menu is brought up.
+  document.addEventListener('click', hideContextMenu);
```

Comment:
Sounds good to me! `domUtil` is a reasonable place to put this part, and the `KeyCode`.